### DISCUSSION TITLE
All power buttons directly on start menu base

### DISCUSSION THREAD

**Discussion by Jamie8848** [ramensoftware/windhawk-mods#1330] [General]
Currently there is a single power button bottom right of the start menu that clicks to multiple options: restart, shutdown, sleep, lock. If those options were directly put along the bottom of the start menu, it would save a click.
I would prefer just the icons as they are self-explanatory, however perhaps some people might want the text function on mouseover.
Also, the option to swap sides for the user name and the power button would mean the power buttons would be a closer mouse journey from the taskbar start button. 

Windhawk is awesome!

**Comment by m417z** [comment]
> the option to swap sides for the user name and the power button would mean the power buttons would be a closer mouse journey from the taskbar start button.

This was demonstrated with the [Windows 11 Start Menu Styler](https://windhawk.net/mods/windows-11-start-menu-styler) mod, and there's this theme which does that: [SideBySide2](https://github.com/ramensoftware/windows-11-start-menu-styling-guide/tree/main/Themes/SideBySide2/README.md).

**Edit**: And a couple of other themes too: [SideBySideMinimal](https://github.com/ramensoftware/windows-11-start-menu-styling-guide/blob/main/Themes/SideBySideMinimal/README.md), [Windows10](https://github.com/ramensoftware/windows-11-start-menu-styling-guide/blob/main/Themes/Windows10/README.md).

**Comment by Jamie8848** [comment]
Thank you, you are way ahead of me.
Windows 11 Start Menu Styler: I was afraid to take a dive into adding my own Target and some Styles but copied and pasted bits from the RosePine theme and it worked.
As a person afraid of even doing the above, I would still love to see a separate Mod that does this with clicking a few options rather than reading through things I have little idea about.
Have a great day!
